l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py

```diff
@@ -229,11 +233,17 @@ def _get_rentas_cordoba_data(self, partner, date, to_date):
         headers = {"content-type": "application/json"}
 
         # Realizar solicitud
-        r = requests.post(url, data=json.dumps(payload), headers=headers, timeout=10)
-        json_body = r.json()
-
-        if r.status_code != 200:
-            raise UserError("Error al contactar rentascordoba.gob.ar. " "El servidor respondió: \n\n%s" % json_body)
+        try:
+            r = requests.post(url, data=json.dumps(payload), headers=headers, timeout=10)
+            json_body = r.json()
+        except requests.exceptions.Timeout:
+            msg = "Timeout error when getting data from rentascordoba.gob.ar"
+            _logger.warning("%s" % msg)
+            raise UserError(_("%s") % msg)
```

Comment:
self.env._

Reply:
check if the text is actually added as a translation term or either the "%s" is only added